docs/guides/bulk/download.md

```diff
@@ -0,0 +1,32 @@
+---
+heading: Bulk
+seo: Bulk | Bulk Download | How to Use Bulk Download
+title: Download
+description: How to use Bulk Download
+layout: sidebarleft
+apis: API Docs
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 2
+---
+
+# Bulk Download
+
+The Bulk download can be used to generate a csv or json file from a large number of records. Since the bulk download works asynchronously, two calls are required to download a file.
+
+### 1. Create a Bulk Job
```

Comment:
+The `POST /bulk/query` creates a bulk job, which builds out a file in Cloud Elements. Use a CEQL query to specify the records included in the file. Since the job will take an unknown amount of time to complete, add a callback url to the bulk job. When the job completes it will make a callback to the url. The **query** is composed of several parts:

The **query**? Are you referring to the API call or the query described below? I think you're referring to the API call.